runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -540,7 +540,7 @@ public void onMenuEntryAdded(MenuEntryAdded menuEntryAdded)
 		// Deposit- op 1 is the current withdraw amount 1/5/10/x
 		if (shiftModifier() && config.bankWithdrawShiftClick() != ShiftWithdrawMode.OFF
 			&& menuEntryAdded.getType() == MenuAction.CC_OP.getId() && menuEntryAdded.getIdentifier() == 1
-			&& menuEntryAdded.getOption().startsWith("Withdraw"))
+			&& (menuEntryAdded.getOption().startsWith("Withdraw") || menuEntryAdded.getOption().equals("Placeholder")))
```

Comment:
This is here to detect top-most op, which is added last, in order to do the swap once. `Placeholder` is never that (it is always the current withdraw mode). So I think you should be able to remove this?

Reply:
The file has been reverted.